projects/public-search/src/app/patron-profile/patron-profile-message/patron-profile-message.component.spec.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2021 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Nl2brPipe } from '@rero/ng-core';
+import { of } from 'rxjs';
+import { PatronApiService } from '../../api/patron-api.service';
+import { PatronProfileMessageComponent } from './patron-profile-message.component';
+
+
+describe('PatronProfileMessageComponent', () => {
+  let component: PatronProfileMessageComponent;
+  let fixture: ComponentFixture<PatronProfileMessageComponent>;
+
+  const messages = [
+    { type: 'success', content: 'Message 1' },
+    { type: 'warning', content: 'Message 2' }
+  ];
+
+  const patronApiServiceSpy = jasmine.createSpyObj('PatronApiService', ['getMessages']);
+  patronApiServiceSpy.getMessages.and.returnValue(of(messages));
+
+  beforeEach(async () => {
```

Comment:
Think `async` is deprected. You should use `waitForAsync` (everywhere) instead

Reply:
done